server/dataGen.js

```diff
@@ -0,0 +1,68 @@
+
+const faker = require('faker');
+const fs = require('fs');
+const createCsvStringifier = require('csv-writer').createObjectCsvStringifier;
+const csvStringifier = createCsvStringifier({
+    header: [
+            {id: 'name', title: 'NAME'},
```

Comment:
@BenChasin Everything else looks good, but something crazy happened with the spacing on this file. Not a priority, but you may want to have ESLINT fix that for you to save some time.